### ISSUE TITLE
2 and 3-digit reference numbers overflow into reference title

### ISSUE THREAD

**Issue by dhimmel** [manubot/rootstock#428]
https://github.com/manubot/rootstock/commit/defe787b0ad8a21a745cc82c3117770896caaf8b updated Pandoc which caused the HTML bibliography styling to change. We also updated our plugins.

Notice the layout of the reference numbers before: https://greenelab.github.io/connectivity-search-manuscript/v/650112ab029fbe6ce1f8acdf65d7d60da03fa49f/#references

![image](https://user-images.githubusercontent.com/1117703/122648295-a5f8a480-d0f6-11eb-8761-37ddd67fc879.png)

And now after (currently): https://greenelab.github.io/connectivity-search-manuscript/v/dd6f847953b86d415d0fa8b45e71923bf5f44fb6/#references

![image](https://user-images.githubusercontent.com/1117703/122648267-70ec5200-d0f6-11eb-98f8-2fef3dabe35b.png)

The new layout where the list numbers are separated into their own column looks nicer, but currently does not adapt well to double digit numbers. For manuscripts with 100+ references, the issue might be even worse. @vincerubinetti is there a way we could accommodate three-digit numbers in the list? And/or have more spacing after the 2-digit numbers?

**Comment by agitter** [comment]
The COVID-19 review has 4 digit reference numbers.  This [HTML artifact](https://ci.appveyor.com/api/buildjobs/n661bftxype3ww3d/artifacts/manuscript-1.0.4150-7475532.html) shows how those look
![image](https://user-images.githubusercontent.com/3532898/122673466-30441580-d196-11eb-9ae8-80cfbcf4ab8c.png)

**Comment by dhimmel** [comment]
Ah that is actually not bad. I was worried the text would overlap. The minimal fix would probably be to ensure that there is at least one space after the number period?

**Comment by vincerubinetti** [comment]
Indeed, I checked to make sure the text wouldn't overlap in that last PR.

A space is already there after the period, but you'd probably want two. Whitespace gets collapsed in HTML, so you'd have to write `&nbsp;&nbsp;`. I'm not sure if that will work in the CSL definition though.

Here is how it looks with two:

![image](https://user-images.githubusercontent.com/8326331/122791216-0ae30480-d287-11eb-85f6-1451154092ac.png)

To avoid the alignment issue, a better solution is to just increase the left margin of the right side of the citation. This would be the style to add (somewhere in `default.html` theme, there may be a section already for references):

```css
.csl-right-inline {
  margin-left: 3em;
}
```

Anywhere from `2.5em` to `3em` looks good to me.

**Comment by dhimmel** [comment]
> To avoid the alignment issue, a better solution is to just increase the left margin of the right side of the citation

Nice. I like that. Did you want to make a PR for this?

**Comment by vincerubinetti** [comment]
I'm on vacation this week, but I can do it on the 28th, or you can do it. It's not critical though, just a minor aesthetic issue.

**Comment by dhimmel** [comment]
Enjoy the vacation 🏖️  & take your time. Not urgent at all.
